### ISSUE TITLE
Support `eslint-plugin-vue`'s disable comments

### ISSUE THREAD

**Issue by FloEdelmann** [mysticatea/eslint-plugin-eslint-comments#75]
In Vue Single File Components (`*.vue` files), "HTML comments" can be used to disable ESLint rules: https://eslint.vuejs.org/rules/comment-directive.html

```vue
<template>
  <!-- eslint-disable-next-line vue/max-attributes-per-line -->
  <div a="1" b="2" c="3" d="4" />
</template>
```

Unfortunately, those comments are not detected by `eslint-plugin-eslint-comments`.

**Comment by MichaelDeBoey** [comment]
Hi @FloEdelmann!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-eslint-comments

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org

**Comment by FloEdelmann** [comment]
Thanks @MichaelDeBoey, I opened https://github.com/eslint-community/eslint-plugin-eslint-comments/issues/136.
